views/templates/admin/marketinge_step4/filters_free.tpl

```diff
@@ -41,13 +40,13 @@
 		<div class="form-group">
 			<div class="title text-left">{l s='Subscription filters :' mod='expressmailing'}</div>
 			<div class="col-lg-12" style="padding-left: 20px">
-				<div class="checkbox">
+				<div class="checkbox" style="float: none">
 					<label for="subscriptions_campaign_optin"><input type="checkbox" {if $subscriptions_campaign_optin}checked="checked"{/if} value="1" class="checkbox-inline" id="subscriptions_campaign_optin" name="subscriptions_campaign_optin">{l s='All Optin customers' mod='expressmailing'} &nbsp; <span class="badge" style="font-size: smaller">{$customers_suscriptions.total_optin|intval}</span></label>
 				</div>
-				<div class="checkbox">
+				<div class="checkbox" style="float: none">
 					<label for="subscriptions_campaign_newsletter"><input type="checkbox" {if $subscriptions_campaign_newsletter}checked="checked"{/if} value="1" class="checkbox-inline" id="subscriptions_campaign_newsletter" name="subscriptions_campaign_newsletter">{l s='All Newsletter suscribed customers' mod='expressmailing'} &nbsp; <span class="badge" style="font-size: smaller">{$customers_suscriptions.total_newsletter|intval}</span></label>
 				</div>
-				<div class="checkbox">
+				<div class="checkbox" style="float: none">
```

Comment:
Here again, we have multiple occurrences of ` style="float: none"`. You should move it in a CSS file.